modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaGetLayoutDisplayPageObjectProviderCheck.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.petra.string.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.check.util.SourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Michael Cavalcanti
+ */
+public class UpgradeJavaGetLayoutDisplayPageObjectProviderCheck
+	extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if (!fileName.endsWith(".java")) {
+			return content;
+		}
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		List<String> importNames = javaClass.getImportNames();
+
+		if (!importNames.contains(
+				"com.liferay.layout.display.page.LayoutDisplayPageProvider")) {
+
+			return content;
+		}
+
+		return _replaceParameterGetLayoutDisplayPageObjectProvider(
+			content, javaClass);
+	}
+
+	private boolean _hasClassNameLong(
+		String content, String fileContent, String variableName) {
+
+		return Objects.equals(
+			getVariableTypeName(content, fileContent, variableName), "long");
+	}
+
+	private String _infoItemReferenceImplementation(
```

Comment:
@michaelmdelcavalcanti, methods are in charge to perform actions, therefore please try to always name them using verbs, in this case, add the prefix `get`. The method should be named as `getInfoItem...`